final/app/src/main/java/com/example/makeitso/model/service/StorageService.kt

```diff
@@ -25,4 +25,8 @@ interface StorageService {
   suspend fun save(task: Task): String
   suspend fun update(task: Task)
   suspend fun delete(taskId: String)
+  suspend fun getCompletedTasksCount(): Long
+  suspend fun getImportantCompletedTasksCount(): Int
```

Comment:
Looks like this is the only `count` declared as an `Int` (others are `Long`) - was it intentional?

Reply:
My mistake! Fixed.